test/unit/plugins/jira.test.js

```diff
@@ -1,14 +1,18 @@
 const assume = require('assume');
+const proxyquire = require('proxyquire');
 const sinon = require('sinon');
-const request = require('request-promise-native');
-
-const JiraPlugin = require('../../../plugins/jira');
+const { json } = require('express');
 
 const sandbox = sinon.createSandbox();
 const addCommentStub = sandbox.stub();
 const commenter = {
   addComment: addCommentStub
 };
+const fetchStub = sandbox.stub();
+
+const JiraPlugin =  proxyquire('../../../plugins/jira', {
+  'node-fetch': fetchStub
+});
 
 const jiraPlugin = new JiraPlugin();
```

Comment:
All of this setup should really be in a `before` or `beforeEach` to avoid running these things unless these tests are being run / avoid leaking things across tests